message/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_library(message STATIC)
+target_sources(message
+  PRIVATE
+    messaging_service.cc
+    messaging_service.hh)
+target_include_directories(message
+  PUBLIC
+    ${CMAKE_SOURCE_DIR})
+target_link_libraries(message
+  PUBLIC
+    gms
+    Seastar::seastar
+  PRIVATE
+    idl)
```

Comment:
messaging_service.o isn't the only object file that depends on idl.

configure.py makes all of them depend on idl, why not here too?

Reply:
yeah. i see you are referencing https://github.com/scylladb/scylladb/blob/2a81b2e49af8ca01a058675dd4b468c6145f9aad/configure.py#L2110 , as `gen_headers` includes `serializers.keys`,see 
https://github.com/scylladb/scylladb/blob/2a81b2e49af8ca01a058675dd4b468c6145f9aad/configure.py#L2102 . so all objects depend on idl. but i add this specific dependency for two reasons:

1. to minimize the dependency whenever possible.
2. probably it's not that relevant here. but i think it will pay off in the longer term -- to make it more correct.
3. it would be convoluted to add a dependency for every single .o file, unless we create a "bootstrap" CMake project.